packages/client/lib/sync/sync.ts

```diff
@@ -168,18 +167,14 @@ export abstract class Synchronizer {
    * @return Resolves with true if sync successful
    */
   async sync(): Promise<boolean> {
-    const peer = this.best()
-    // TODO: only activate along fixing test failures
-    /*let numAttempts = 1
-    while (!peer) {
-      if (numAttempts === 2) {
-        this.syncTargetHeight = this.chain.headers.height
-        this.updateSynchronizedState()
-      }
+    let peer = this.best()
+    let numAttempts = 1
+    while (!peer && this.opened) {
+      this.config.logger.debug(`Waiting for best peer (attempt #${numAttempts})`)
       await new Promise((resolve) => setTimeout(resolve, 5000))
       peer = this.best()
       numAttempts += 1
-    }*/
+    }
```

Comment:
Oh, interesting. This can now be reactivated without breaking the tests? Might have accidentally fixed something on a more profound level when reworking some initialization parts of the sync and service tests. 😋